modular_skyrat/modules/contractor/code/items/misc.dm

```diff
@@ -60,3 +60,49 @@
 
+
+/obj/item/paper/contractor_guide/midround
+	name = "Contractor Guide"
+
+/obj/item/paper/contractor_guide/midround/Initialize(mapload)
+	info = {"<p>Welcome agent, congratulations on successfully getting in range of the station.</p>
+
+			<p>You likely already have your Contractor MODSuit equipped. It has a built in chameleon module, which only works when the MODSuit is undeployed,
+			but is highly useful for on-station infiltrations. We also provide your chameleon jumpsuit and mask, both of which can be changed
+			to any form you need for the moment. The cigarettes are a special blend - it'll heal your injuries slowly overtime.</p>
+
+			<p>Your standard issue contractor baton can be found in the baton holster MODSuit module, it hits harder than the ones you might be used to,
+			and will likely be your go to weapon for kidnapping your targets. We took the liberty of fully upgrading it beforehand, so don't worry about that.
+			The three additional items have been randomly selected from what we had available. We hope they're useful to you for your mission.</p>
+
+			<p>The contractor hub, available at the top right of the uplink, will provide you unique items and abilities. These are bought using Contractor Rep,
+			with two Rep being provided each time you complete a contract.</p>
+
+			<h3>Using the tablet</h3>
+			<ol>
+				<li>Open the Syndicate Contract Uplink program.</li>
+				<li>Here, you can accept a contract, and redeem your TC payments from completed contracts.</li>
+				<li>The payment number shown in brackets is the bonus you'll receive when bringing your target <b>alive</b>. You receive the
+				other number regardless of if they were alive or dead.</li>
+				<li>Contracts are completed by bringing the target to designated dropoff, calling for extraction, and putting them
+				inside the pod.</li>
+			</ol>
+
+			<p>Be careful when accepting a contract. While you'll be able to see the location of the dropoff point, cancelling will make it
+			unavailable to take on again.</p>
+			<p>The tablet can also be recharged at any cell charger.</p>
+			<h3>Extracting</h3>
+			<ol>
+				<li>Make sure both yourself and your target are at the dropoff.</li>
+				<li>Call the extraction, and stand back from the drop point.</li>
+				<li>If it fails, make sure your target is inside, and there's a free space for the pod to land.</li>
+				<li>Grab your target, and drag them into the pod.</li>
+			</ol>
+			<h3>Ransoms</h3>
+			<p>We need your target for our own reasons, but we ransom them back to your mission area once their use is served. They will return back
+			from where you sent them off from in several minutes time. Don't worry, we give you a cut of what we get paid. We pay this into whatever
+			ID card you have equipped, on top of the TC payment we give.</p>
+
+			<p>Good luck agent. You can burn this document with the supplied lighter.</p>"}
```

Comment:
What even is different here from the other one? That's a *lot* of repetition for something that doesn't seem to change much.

Also that formatting is really strange.

Reply:
Mostly the first few paragraphs were changed.